### PR TITLE
DTSERWTHRE 85 prepare translation for transfer to bank card

### PR DESCRIPTION
**Pull request by aparthibanpaypal**
Ready for review

### CODE REVIEW

File: Sources/Model/Receipt/HyperwalletReceipt.swift [hunk 1]
```diff
@@ -395,6 +395,8 @@ public struct HyperwalletReceipt: Decodable, Equatable {
         case venmoAccountTransferReturn = "VENMO_ACCOUNT_TRANSFER_RETURN"
         /// Default - unknown transfer type
         case unknown = "UNKNOWN_RECEIPT_TYPE"
+        /// Transfer To Bank Card
+        case transferToBankCard = "TRANSFER_TO_BANK_CARD"
```
Comment (ddhar-hw) [inline Sources/Model/Receipt/HyperwalletReceipt.swift:399]: I would suggest to keep `unknown` case at the end

**Review by jsaini-hw — CHANGES_REQUESTED** [review]
Fix @ddhar-hw 's code review comment

File: Sources/Model/Receipt/HyperwalletReceipt.swift (hunk 1 above)
Comment (aparthibanpaypal) [reply Sources/Model/Receipt/HyperwalletReceipt.swift:399]: Sure will update

**Review by nanbehera — APPROVED** [review]
approved

**Review by ddhar-hw — APPROVED** [review]

**Review by jsaini-hw — APPROVED** [review]

**Pull request merged**